app/assets/stylesheets/layout_components/gift-cards.scss

```diff
@@ -0,0 +1,109 @@
+.reward-card-image{
+  /* Foundation equalizer needs a height for images. */
+  height: 196px;
+}
+
+.reward-card-title{
+  background: $primary-color;
+  padding: 0 $base-padding*4;
+
+  h4{
+    color: $white;
+  }
+}
+
+.reward-card-body{
+  background: $secondary-color;
+  padding: 0 $base-padding*4;
+
+  p,
+  label{
+    color: $white;
+    float: left;
+  }
+}
+
+.card-balance{
+  font-size: 2rem;
+  line-height: 2rem;
+  color: $white;
+}
+
+.card-number{
+  margin: 0 0 0 $base-margin*3;
+}
+
+.iframe-input{
+  margin-bottom: $base-margin !important;
+}
+
+.secure{
+  width: 100%;
+
+  img{
+    padding-left: $base-padding*2;
+    margin-top: -$base-margin/2;
+  }
+}
+
+.clickable-radio-buttons{
+  input[type="radio"] + label{
+    background: $white;
+    color: $secondary-color;
+    border-radius: $base-radius;
+    border: none;
+    margin: 0 $base-margin 0 0;
+    padding: $base-padding/3;
+    width: 19%;
+    text-align: center;
+    float: left;
+
+    &:before,
+    &:after{
+      content: none;
+    }
+  }
+
+  input[type="radio"]:checked + label,
+  input[type="radio"]:hover + label{
+    background: $primary-color;
+    color: $white;
+    border: none;
+  }
+
+  .small-link{
+    color: $white;
+    font-size: 0.7rem;
+    line-height: 15px;
+    display: block;
+  }
+  @include breakpoint(medium down){
+    input[type="radio"] + label{
+      padding: $base-padding $base-padding/2;
+      margin-bottom: $base-margin;
+    }
+  }
+}
+
+.toggle-card{
+  color: $white;
+  font-size: 0.7rem;
+  line-height: 15px;
+  display: block;
+  margin-bottom: $base-margin;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+#toggle-amount{
```

Comment:
Most of the styles here match the ones in `.toggle-card` above. Should the common parts be moved into an includable mixin, or combine the common rules `.toggle-card, #toggle-amount{ ... }`, or is a bit of repetition fine?